homeassistant/components/fritz/quality_scale.yaml

```diff
@@ -0,0 +1,96 @@
+rules:
+  # Bronze
+  action-setup:
+    status: todo
+    comment: still in async_setup_entry, needs to be moved to async_setup
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage:
+    status: todo
+    comment: one coverage miss in line 110
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions:
+    status: todo
+    comment: include the proper docs snippet
+  entity-event-setup: done
+  entity-unique-id: done
+  has-entity-name:
+    status: todo
+    comment: partially done
+  runtime-data:
+    status: todo
+    comment: still uses hass.data
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
+
+  # Silver
+  action-exceptions: done
+  config-entry-unloading: done
+  docs-configuration-parameters: done
+  docs-installation-parameters:
+    status: todo
+    comment: add the proper configuration_basic block
+  entity-unavailable: done
+  integration-owner: done
+  log-when-unavailable: done
+  parallel-updates:
+    status: todo
+    comment: not set at the moment, we use a coordinator
+  reauthentication-flow: done
+  test-coverage:
+    status: todo
+    comment: we are at 88% coverage at the moment
+
+  # Gold
+  devices: deno
+  diagnostics: done
+  discovery-update-info: todo
+  discovery: done
+  docs-data-update: todo
+  docs-examples: done
+  docs-known-limitations:
+    status: exempt
+    comment: no known limitations, yet
```

Comment:
Do we want to mention there are no known limitations? Is there something you would expect the device to do but it doesnt?

Reply:
you mean adding a section "Known limitations" to the docs, with just the sentence "There are no known limitations, yet" ?

Reply:
I think that will add to consistency of the docs. It is nice that the docs are complete, so you know you can always find something there

Reply:
In respect to consistency, I think we should have a skeleton for docs :thinking:

Reply:
That also sounds like a good idea

Reply:
btw. be compliant to this skeleton could also be a quality scale rule afterwards :grimacing:

Reply:
I think that is another discussion. I do believe there is a line between consistency and freedom, because not every integration would follow the same order. I can imagine an integration like switchbot (where every device has its own platform ish (as in, curtain -> cover, fingerbot -> button, etc), would have a different setup (maybe having the docs device central is more useful there than just having everything per category)